.vsts-dotnet-ci.yml

```diff
@@ -4,8 +4,11 @@
 
 resources:
   containers:
-  - container: LinuxContainer
+  - container: LinuxCentosContainer
     image: microsoft/dotnet-buildtools-prereqs:centos-7-b46d863-20180719033416
+  
+  - container: LinuxUbuntuContainer
+    image: microsoft/dotnet-buildtools-prereqs:ubuntu1604_cross_prereqs_v1
```

Comment:
From the test failures, it looks like this image doesn't have `libgdiplus` installed on it.